exporters/zipkin/src/recordable.cc

```diff
@@ -0,0 +1,184 @@
+#include "opentelemetry/exporters/zipkin/recordable.h"
+
+#include <iostream>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace zipkin
+{
+
+const int kAttributeValueSize = 14;
+
+void Recordable::SetIds(trace::TraceId trace_id,
```

Comment:
Is this function reusable by other exporters, would they use the same field names? `id` , `parentId`, `traceId` - should this be moved into common utility function within the SDK?

@mishal23 - you need to borrow this same code for ETW exporter, to populate the actual IDs in your PR.

Reply:
@maxgolov  - Not sure if we can move it to common. As different exporters expect these fields in different formats. E.g, `Zipkin` as hex chars of 16/32 length, `Jaeger` as uint64_t.

Reply:
The implementation of `Recordable` is exporter specific. As exporter could choose different underlying storage, agree that it is unnecessary to provide a common implementation.